### ISSUE TITLE
Feature: Refresh the branding for the Toolkit

### ISSUE THREAD

**Issue by Lamparter** [RiversideValley/Toolkit#2]
### 📄 Description

The new super-merged toolkit is in desperate need of new branding and representation. Since it combines other toolkits, there needs to be a unified name to represent it (I can't go on longer saying just "the Toolkit"). There also needs to be a new logo as well.

### 🗃️ Alternative solutions

> [!NOTE]
> If you have any name ideas or logo suggestions please make a comment!

**Name ideas**:
- ~~OneToolkit~~
> won't work because I want to make @JaiganeshKumaran happy
- ~~`Riverside.Toolkit`~~
> ugly name
- Crimson Toolkit
> nice but doesn't have a good ring to it
- ~~GlowUI Toolkit~~
> I want to make sure GlowUI and this toolkit are very different (although the toolkit contains it, the toolkit is more since it contains CubeKit, Riverside.Toolkit and more)

### ⚡ API Changes

Won't affect namespace since that will always be the same

### ✅ Tasks

```[tasklist]
### High Priority
- [ ] Something
- [ ] Another thing
- [ ] https://github.com/link/to/an/issue
```
```[tasklist]
### Nice to have
- [ ] Something
- [ ] Another thing
- [ ] https://github.com/link/to/an/issue
```

### 📸 Assets

_None yet_

**Comment by Lamparter** [comment]
The name: 
- CubeKit
- `Riverside.Toolkit` as a namespace
The logo hasn't been designed yet but I imagine a re-colouring of the WinUI3 logo would be good, like what was done for CrimsonUI back in the day.

**Comment by FireCubeStudios** [comment]
Cubekit?

**Comment by IviriusMain** [comment]
Yes this is great for combining parts from each of our toolkits into one.
And about the logo I'll think of something.
